explorer/explorer.go

```diff
@@ -616,14 +621,16 @@ func (exp *explorerUI) Store(blockData *blockdata.BlockData, msgBlock *wire.MsgB
 		go exp.updateDevFundBalance()
 	}
 
-	// Do not run updates if blockchain sync is running.
-	if !exp.AreDBsSyncing() {
+	// Do not run updates if blockchain sync is running or lite mode is on.
+	if !exp.AreDBsSyncing() && !exp.liteMode {
 		// Update the charts data after every five blocks or if no charts data
 		// exists yet.
+
 		if newBlockData.Height%5 == 0 || cacheChartsData.Height() == -1 {
```

Comment:
Changes made after every block are too small to be visible on the charts since most of them hold data running into thousands of entries.
With this PR it takes 1-3 sec to fetch the incremental updates. Imagine a case scenario where you spend 2 sec on average per block to fetch data that a user might not easily notice. An interval of 5 blocks ensures that the other 8 sec are spent doing other things instead of querying the charts data.

Reply:
I'm okay with 5 block increments. Charts can be zoomed though, so there aren't always going to be thousands of data points displayed.